plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-product-variations-controller.php

```diff
@@ -65,6 +65,30 @@ public function register_routes() {
 		);
 	}
 
+	/**
+	 * Get the downloads for a product variation.
+	 *
+	 * @param WC_Product_Variation $product Product variation instance.
+	 * @param string               $context Context of the request: 'view' or 'edit'.
+	 *
+	 * @return array
+	 */
+	protected function get_downloads( $product, $context = 'view' ) {
+		$downloads = array();
+
+		if ( $product->is_downloadable() || 'edit' === $context ) {
```

Comment:
Shouldn't this be `&&` instead of `||`?

Reply:
Nope! Because it should matches when downloadable is `false` and the context is `edit`. I need the `downloads` files even when the product is not downloadable.

Reply:
Thank you for clarifying 🙂